arcgis10_mapping_tools/MapAction/MapAction/Utilities.cs

```diff
@@ -193,33 +224,21 @@ public static string createXML(Dictionary<string, string> usDict, string rootEle
 
             cmfURI = new Uri(System.IO.Path.GetDirectoryName(opCfgFilePath), UriKind.Absolute);
 
-            //If the file exists in the filepath, add each element and value of the xml file 
-            //to the dictionary as key value pairs 
             try
             {
                 if (File.Exists(@opCfgFilePath))
                 {
-                    XDocument doc = XDocument.Load(@opCfgFilePath);
-                    foreach (XElement usEle in doc.Root.Descendants())
-                    {
-                        if (usEle.Name.ToString().Equals("DefaultPathToExportDir", StringComparison.InvariantCultureIgnoreCase))
-                        {
-                            //Uri absExportURI = new Uri(usEle.Value.ToString(), UriKind.Absolute);
-                            //absExportURI.MakeRelativeUri(cmfURI)
-                            dict.Add(usEle.Name.ToString(), absPathFromRel(usEle.Value.ToString()));
-                        }
-                        else
-                        {
-                            dict.Add(usEle.Name.ToString(), usEle.Value.ToString());
-                        }
-                    }
+                    XmlSerializer serializer = new XmlSerializer(typeof(OperationConfig));

```

Comment:
Notwithstanding my prev comment i love this approach of using the XmlSerializer - it's not something I've used before though so I'm going to take your implementation on trust and learn from it 👍